### PR TITLE
a30-New-York-Times

### PR DESCRIPTION
**Pull request by valentinavargas31**

### CODE REVIEW

File: src/mm/A30-New-York-Times/styles.css [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  font-size: 20px;
+  background-color: rgb(250, 247, 247);
+}
+
+p {
+  font-size: 20px;
+  margin-right: 60px;
+}
+
+.options nav a {
```
Comment (Iregui44) [inline src/mm/A30-New-York-Times/styles.css:86]: I think options should cover a little more of the page

File: src/mm/A30-New-York-Times/index.html [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+          <div class="links-header">
+            <a href="" id="Sign">SIGN IN</a>
+            <a href="" id="subscribe">SUBSCRIBE</a>
+          </div>
+        </div>
+        <hr class="header-separator" />
+        <div class="options">
+          <p><strong>Thursday</strong> <br />May 21, 2020</p>
+          <nav>
+            <a href="">World</a>
```
Comment (Iregui44) [inline src/mm/A30-New-York-Times/index.html:48]: This are not suppposed o be buttons? also add "#" to href when there is not an existing link.

File: src/mm/A30-New-York-Times/styles.css [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+  border: none;
+  font-weight: bold;
+}
+
+#subscribe {
+  background-color: rgb(250, 247, 247);
+  color: #000000;
+  border: solid rgb(247, 244, 244);
+}
+.options {
```
Comment (Iregui44) [inline src/mm/A30-New-York-Times/styles.css:74]: I think this options should cover more of the page.

File: src/mm/A30-New-York-Times/styles.css [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+
+.header-separator {
+  border: none;
+  border-bottom: 1px solid #e5e5e5;
+}
+
+.button-1 {
+  background-color: #000000;
+  color: #ffffff;
+  padding: 20px 20px 20px 20px;
```
Comment (Iregui44) [inline src/mm/A30-New-York-Times/styles.css:100]: Yo can use padding: 20px if all values are the same

**Comment by Iregui44** [comment]
I dont see any corrections

File: src/mm/A30-New-York-Times/styles.css (hunk 1 above)
Comment (valentinavargas31) [reply src/mm/A30-New-York-Times/styles.css:86]: corrected

File: src/mm/A30-New-York-Times/index.html (hunk 2 above)
Comment (valentinavargas31) [reply src/mm/A30-New-York-Times/index.html:48]: I changed them to buttons

File: src/mm/A30-New-York-Times/styles.css (hunk 3 above)
Comment (valentinavargas31) [reply src/mm/A30-New-York-Times/styles.css:74]: ok

File: src/mm/A30-New-York-Times/styles.css (hunk 4 above)
Comment (valentinavargas31) [reply src/mm/A30-New-York-Times/styles.css:100]: corrected

**Comment by valentinavargas31** [comment]
corrected page

File: src/mm/A30-New-York-Times/styles.css (hunk 4 above)
Comment (Iregui44) [reply src/mm/A30-New-York-Times/styles.css:105]: where?
```suggestion
  padding: 20px;
```

**Comment by Iregui44** [comment]
You are missing corrections

File: src/mm/A30-New-York-Times/index.html [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+            <button id="sign">SIGN IN</button>
+            <button id="subscribe">SUBSCRIBE</button>
+          </div>
+        </div>
+        <hr class="header-separator" />
+        <div class="options">
+          <p><strong>Thursday</strong> <br />May 21, 2020</p>
+          <nav>
+            <button>World</button>
+            <button>Politics</button>
```
Comment (Iregui44) [inline src/mm/A30-New-York-Times/index.html:50]: buttons must have href

File: src/mm/A30-New-York-Times/index.html [hunk 6]
```diff
@@ -0,0 +1,214 @@
...
+            <a href="">Science</a>
+            <a href="">Sports</a>
+            <a href="">Arts</a>
+            <a href="">Books</a>
+            <a href="">Style</a>
+            <a href="">Food</a>
+            <a href="">Travel</a>
+            <a href="">Magazine</a>
+            <button class="button-1">
+              <i class="fa-solid fa-ellipsis"></i>
```
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/A30-New-York-Times/index.html]: corrected

File: src/mm/A30-New-York-Times/styles.css [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  background-color: rgb(250, 247, 247);
+  font-family: serif;
+}
+
+.sidebar {
```
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/A30-New-York-Times/styles.css:11]: corrected

File: src/mm/A30-New-York-Times/index.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (valentinavargas31) [reply src/mm/A30-New-York-Times/index.html]: ok

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**